datastore/longhorn.go

```diff
@@ -226,12 +226,23 @@ func (s *DataStore) GetSettingValueExisted(sName types.SettingName) (string, err
 	if err != nil {
 		return "", err
 	}
-	if setting.Value == "" {
```

Comment:
Cannot remove this. This function guarantees if the setting is empty it will error out, that's why it's called `GetSettingValueEXISTED`